lib/fluent/load.rb

```diff
@@ -15,6 +15,15 @@
   # ignore setup error on Win or similar platform which doesn't support signal
 end
 require 'cool.io'
+
+$platformwin = /mswin|mingw/ === RUBY_PLATFORM
+if $platformwin
+  require_relative 'winfile'
+else
+  module Fluent
+    Win32File = File
```

Comment:
How about using `FileWrapper` or something name?
Using `Win32File` in everywhere seems to misread `this code doesn't work on Linux?`